tests/testthat/test-Model-validity.R

```diff
@@ -1079,14 +1079,14 @@ test_that("v_model_one_par_exp_normal_prior returns message for wrong skel_probs
   object@skel_probs <- c(-1, 0.5, Inf)
   expect_equal(
     v_model_one_par_exp_normal_prior(object),
-    "skel_probs must be sorted probability values between 0 and 1"
+    "skel_probs must be a unique sorted probability values between 0 and 1"
   )
 
   # Assigning not sorted skel_probs.
   object@skel_probs <- c(0.2, 0.1)
```

Comment:
what happens with `c(0.2, 0.2)`? is that caught?

Reply:
Unfortunately not yet, as I have to enhance `assert_probabilities` to be able to check for the uniqueness. I just prepared these error messages already.

Reply:
but maybe we can actually allow repeated values. The function will not be strictly incr. then, but since we have this interpolation - then it can be fine still. What do you think @danielinteractive ? Dose it make sense practically, that we have the same DLT prob for different doses?

Reply:
yeah but then it is not a bijective function?

Reply:
Yeah, that is right, sorry I do not why I though that this wont be a problem with this interpolation. But this is obviously  wrong. So we have to add it.

Reply:
Would it be ok if you accept it so that these unit tests errors disappear from the `main` and then I will work on the `assert_probabilities`? Tx!

Reply:
ok, just make a new issue for this then

Reply:
Sure, thanks. Created - https://github.com/Roche/crmPack/issues/399